pipelines/nextflow/workflows/additional_seq_prepare/nextflow.config

```diff
@@ -19,7 +19,7 @@ includeConfig '../nextflow.config'
 workDir = "$NXF_WORK/additional_seq_prepare"
 
 params {
-    outdir = "$workDir"
+    output_dir = "Output_AdditionPrepare"
```

Comment:
A naive question, does this publish the output in the working directory or from where we are running the nextflow command ?